packages/types/src/authentication.ts

```diff
@@ -1,7 +1,36 @@
 import * as z from 'zod';
+import { UserSummary } from '.';
 
-export const SignUpResponseBody = z.object({
-    userID: z.string().uuid(),
-    userNickname: z.string(),
+export const AuthenticationModeValues = z.enum(['web', 'api']);
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 praise:** ‏Nice! I like these denominations.